config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -1,5 +1,42 @@
 # this file should contain all periodic jobs that use the fejta-bot token
 periodics:
+- name: periodic-api-review-help
+  interval: 1h
+  decorate: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      command:
+      - /commenter
+      args:
+      - |-
+        --query=org:kubernetes
+        is:pr
+        base:master
+        label:kind/api-change
+        -label:api-review
+        -label:do-not-merge/work-in-progress
+        -label:do-not-merge/hold
+        -label:needs-rebase
+      - --updated=5m
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=This PR [may require API review](https://git.k8s.io/community/sig-architecture/api-review-process.md#what-apis-need-to-be-reviewed).
```

Comment:
This needs comment needs to do something which causes it to not match the search the next time it runs an hour later.

Otherwise fejta-bot will leave a duplicate of this comment on the issue every hour.

Aka the restester leaves a /retest comment, which causes it to no longer match that failing status field. The stale job leaves a /lifecycle stale, which causes it to not match the next time around, etc.

Reply:
Ah, good to know. If it excluded based on a bit of the text in the comment itself, would that work? E.g. `… -"api-review"`

Reply:
Depends on what github does with the search string. The commenter is extremely dumb/straight-forward. It  sends your `--query` string (appending `is:closed` by default) to the github search API and leaves the comment on every issue it gets back, every time (stopping at the configured `--ceiling` if necessary). 

https://github.com/kubernetes/test-infra/blob/6f40a71acfc801ca4c1f4a314f2293398487f576/robots/commenter/main.go#L198

https://github.com/kubernetes/test-infra/blob/6f40a71acfc801ca4c1f4a314f2293398487f576/prow/github/client.go#L1864

Reply:
verified adding `NOT "complete the pre-review checklist and request an API review"` excludes PRs the bot has already commented on